client/my-sites/controller.js

```diff
@@ -223,7 +223,7 @@ function onSelectedSiteAvailable( context, basePath ) {
 	const selectedSite = getSelectedSite( state );
 
 	const isAtomicSite = isSiteAutomatedTransfer( state, selectedSite.ID );
-	const userCanManagePlugins = canCurrentUser( state, selectedSite.ID, 'manage_options' );
+	const userCanManagePlugins = canCurrentUser( state, selectedSite.ID, 'activate_plugins' );
```

Comment:
Does it make more sense to use `install_plugins` instead? I'm asking because this is used to redirect to the `plugin-install` screen, and accessing that one requires the `install_plugins` capability.

Reply:
as @kbrown9 pointed out in the internal repo, this is the capability that [core uses](https://core.trac.wordpress.org/browser/trunk/src/wp-admin/menu.php#L240) when adding the plugins menu, so I think we should go with that.